src/xcode/ENA/ENA/Source/HTTPClient/Resources/DCCRules/DCCRulesResource.swift

```diff
@@ -22,6 +22,10 @@ struct DCCRulesResource: Resource {
 			self.type = .caching(
 				Set<CacheUsePolicy>([.loadOnlyOnceADay])
 			)
+		} else if ruleType == .invalidation {
+			self.type = .caching(
+				Set<CacheUsePolicy>([.loadOnlyOnceADay])
+			)
```

Comment:
I think this should be injected as we don't want to change the caching policy for travel validation, or do we?

Reply:
@nickguendling  we have the Else statement which will be executed for the Acceptance rules, or did you mean something else ?

Reply:
```
		if ruleType == .boosterNotification {
			self.type = .caching(
				Set<CacheUsePolicy>([.loadOnlyOnceADay])
			)
		} else if ruleType == .invalidation {
			self.type = .caching(
				Set<CacheUsePolicy>([.loadOnlyOnceADay])
			)
		} else {
			self.type = .caching()
		}

```

Reply:
`HealthCertificateValidationService` is using the same resource via `RulesDownloadService` to download the invalidation rules, so you changed the caching mechanism for the health certificate validation here. I think that is not intended.

Reply:
@nickguendling i pushed again